packages/terra-paginator/src/Paginator.jsx

```diff
@@ -33,6 +33,19 @@ const propTypes = {
 };
 
 class Paginator extends React.Component {
+  static getDerivedStateFromProps(props, state) {
```

Comment:
With using `getDerivedSateFromProps`, we need to polyfill it for react 16.2 since that's our lower bound version of react. To do this, you can [import a polyfill from react-lifecycles-compat](https://github.com/cerner/terra-core/blob/master/packages/terra-form-select/src/_Menu.jsx#L4) and [wrap the paginator component like we do in the select component code](https://github.com/cerner/terra-core/blob/master/packages/terra-form-select/src/_Menu.jsx#L285-L289).

Then you'll want to add the react-lifecycles-compat modules as a dependency to this package [similar to terra-select](https://github.com/cerner/terra-core/blob/master/packages/terra-form-select/package.json#L35).